gen/gen_sim.py.j2

```diff
@@ -42,3 +42,14 @@ class {{ module_name }}{% if inherits is defined %}({{ inherits }}){% endif %}:
 
   {% endif %}
   {% endfor %}
+
+  {% if module_name == 'MotController' %}
```

Comment:
This isn't the right way to do this. Add a 'sim_class_extra' key to MotController_data.yml and put this in the yml file. (see `sim_extra` above)